docs/developer/discounts/displaying-promo-prices.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Displaying Promotional Prices
+sidebar_label: Displaying Promotional Prices
+---
+
+Saleor API provides pricing information that can be consumed by your storefront for four different scenarios:
+- Individual `ProductVariant` pricing, which provides prices of a specific SKU (`VariantPricingInfo`)
+- Aggregated `Product` pricing, which provides a range of `ProductVariant` prices (`ProductPricingInfo`)
+- Pricing information included in `CheckoutLine` for products that are added to the cart/checkout
+- `OrderLine` pricing information concerning products that were purchased by the customer
+
+:::note
+Only [catalogue promotions](developer/discounts/promotions) will result in changes to Product/Variant pricing. 
+Order discounts and vouchers are applied only after the product is added to the Checkout.
+:::
+
+### Types of Promotional Prices
+
+There are three main types of promotional prices that you can use to display prices in a storefront:
+- `price` - includes relevant discounts (depending on context)
+- `priceUndiscounted` - price before any discounts are applied 
+- `pricePrior` - price in the period before the promotion. This price is not tracked automatically by Saleor (see section below)
+
+On top of that, Saleor provides discount calculations relative to `priceUndiscounted` and `pricePrior` for `Product` and `ProductVariant`.
+
+### Using `pricePrior` as a Base for Discount Calculations
+
+For some geographies, it might be required to calculate discounts based on the lowest price before the promotion. 
+An example of such a requirement is the EU Omnibus directive. 
+Given that there are many different interpretations of the "lowest price", Saleor does not track `pricePrior` automatically. 
+However, it can be provided using 
+[`productVariantChannelListingUpdate`](../../api-reference/discounts/mutations/product-variant-channel-listing-update) or 
```

Comment:
Hey @delemeator the links in Docusaurus have to follow a pattern of with the file extension at the end, as stated here: https://github.com/saleor/saleor-docs?tab=readme-ov-file#creating-links

You can also run `npm run build` to run a full build locally, that will show you broken links, that are breaking the Vercel build here 🙌